apps/core-app/src/pages/UpdateSettings.tsx

```diff
@@ -0,0 +1,31 @@
+import { FormBuilder } from '@daohaus/haus-form-builder';
+import { useDao } from '../contexts/DaoContext';
+import { CustomFields } from '../legos/config';
+import { FORM } from '../legos/form';
+
+export function UpdateSettings() {
+  const { dao } = useDao();
+  const links = dao?.links && JSON.parse(dao?.links);
```

Comment:
Wonder if this should be in a use memo or useEffect incase dao is undefined while it fetches

Reply:
Can also avoid the memo or effect by short circuting with a unary

```const links = dao && ...```

Reply:
i added useMemo - should work fine when the dao appears. still need loading state but I think thats for another day.